manifests/api.pp

```diff
@@ -0,0 +1,55 @@
+# == Class: aptly::api
+#
+# Install and configure Aptly's API Service
+#
+# === Parameters
+#
+# [*ensure*]
+#   Ensure to pass on to service type
+#   Default: running
+#
+# [*user*]
+#   User to run the service as.
+#   Default: root
+#
+# [*group*]
+#   Group to run the service as.
+#   Default: root
+#
+# [*listen*]
+#   What IP/port to listen on for API requests.
```

Comment:
Possibly worth documenting that the `:` is needed. Since its requirement suprised me or least link to the aptly docs.